ironfish-cli/src/commands/wallet/burn.ts

```diff
@@ -42,6 +42,12 @@ export class Burn extends IronfishCommand {
       default: false,
       description: 'Confirm without asking',
     }),
+    confirmations: Flags.integer({
```

Comment:
this confirmation flag is very confused with confirm flag in the same command. Can you possibly give a different name? eg min blocks

Reply:
Confirmations is a specific crypto thing, which is why we use this name across the whole code base including CLI commands, arguments, and in our code.